drivers/telescope/lx200am5.h

```diff
@@ -104,6 +104,14 @@ class LX200AM5 : public LX200Generic
             High
         };
 
+        // Heavy duty control
+        INDI::PropertySwitch HeavyDutyModeSP {2};
+        enum
+        {
+            HeavyDutyModeOff = 0,
+            HeavyDutyModeOn  = 1,
+        };
```

Comment:
For such toggles, it's better to remove the enum and use INDI_ENABLED (0) and INDI_DISABLED (1) which are used for other switches. It just makes all the enable/disable switches all consistent for that first element is always "Enabled" and 2nd element "Disabled".

Reply:
ok I will correct it